dagobah/core/dagobah.py

```diff
@@ -19,6 +19,7 @@ class Dagobah(object):
     instance, as well as top-level parameters such as the
     backend used for permanent storage.
     """
+    JIJ_DELIM = '%_|JIJ_DELIMITER|_%'
```

Comment:
the other issue is the default "no name" option is to name it the same as the command, and while rare, I could see `%_|` being found in a command (thinking regexes, delimeter based commands, etc)

I'm wondering now if a convenience method for jobs to return a `readable_name` that would replace the complex delimiter with a readable delimiter would be nice. logs would use this attribute instead of a raw name